app/index.js

```diff
@@ -1,5 +1,27 @@
 import React from 'react'
+import styled from 'styled-components'
 import ReactDOM from 'react-dom'
 import App from './app'
+import { version, date } from './version.json'
 
-ReactDOM.render(<App />, document.getElementById('root'))
+/*
+The Footer component is a temporary stop-gap while we are still
+undergoing heavy development to ensure we know what we are looking at!
+*/
+const Footer = styled.div`
+  margin: 40px;
+  text-align: right;
+  color: DarkGrey;
+  font-size: 14px;
+  border-top: 1px solid LightGrey;
```

Comment:
Better to put this at the end of the `Routes` component (outside the `Switch`). This is the entry point file that mounts the app and it's confusing (to me) to have other components here.

Reply:
I thought it best to be here and out-of-place so people know that its temporary and should get removed.

Reply:
... of course there is the fact it could become a permanent feature :thinking: